config/environments/test.rb

```diff
@@ -51,4 +51,7 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Configure active job to use sneakers/rabbitmq backend
```

Comment:
Comment is misleading

Reply:
Fixed it (only the comment) by commit amend.